Sources/Markdown/Parser/BlockDirectiveParser.swift

```diff
@@ -101,8 +101,8 @@ struct PendingBlockDirective {
             // There may be garbage after the left parenthesis `(`, but we'll
             // still consider subsequent lines for argument text, so we'll
             // indicate acceptance either way at this point.
-            _ = parseArgumentsText(from: line)
             endLocation = line.location!
```

Comment:
Do we still need this line? If `endLocation` is set in `parseArgumentsText(from:)`, won't that override the value set here?